Calibration/LumiAlCaRecoProducers/python/ALCARECOAlCaPCCRandomFromRECO_cff.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+import HLTrigger.HLTfilters.hltHighLevel_cfi
+ALCARECORandomFromRECOHLT = HLTrigger.HLTfilters.hltHighLevel_cfi.hltHighLevel.clone(
+    HLTPaths = cms.vstring("*Random*"),
+    eventSetupPathsKey='',
```

Comment:
hello @capalmer85 

@trtomei and @tocheng 
are in the process of configuring every alcareco to be steerable via triggerbits  for the HLT selection they do, which will ease changes in operations if you'll want to tune the input to your producer.
Please keep in mind we'll want to do that once this PR is merged
( let's not do it now not to couple two rather complex PR's)